models/topics.models.js

```diff
@@ -143,3 +146,20 @@ exports.selectAllUsers = () => {
     return rows;
   });
 };
+
+exports.insertArticle = (author, title, body, topic, article_img_url)=>{
+  return db
+  .query(
+    `INSERT INTO articles(author, title, body, topic, article_img_url) 
+    VALUES ($1, $2, $3, $4, $5) 
+    RETURNING *
+    ;`,
+    [author, title, body, topic, article_img_url]
+
+    
```

Comment:
empty lines for no reason here - make a habit of keeping your formatting consistent